CHANGELOG.md

```diff
@@ -4,6 +4,68 @@
+* [#8100](https://github.com/timescale/timescaledb/pull/8100) Support splitting compressed chunks
+* [#8131](https://github.com/timescale/timescaledb/pull/8131) Add policy to process hypertable invalidations
+* [#8141](https://github.com/timescale/timescaledb/pull/8141) Add function to process hypertable invalidations
+* [#8165](https://github.com/timescale/timescaledb/pull/8165) Reindex recompressed chunks in compression policy
+* [#8178](https://github.com/timescale/timescaledb/pull/8178) Add columnstore option to `CREATE TABLE WITH`
+* [#8179](https://github.com/timescale/timescaledb/pull/8179) Implement direct `DELETE` on non-segmentby columns
+* [#8182](https://github.com/timescale/timescaledb/pull/8182) Cache information for repeated upserts into the same compressed chunk
+* [#8187](https://github.com/timescale/timescaledb/pull/8187) Allow concurrent Continuous Aggregate refreshes
+* [#8191](https://github.com/timescale/timescaledb/pull/8191) Add option to not process hypertable invalidations
+* [#8196](https://github.com/timescale/timescaledb/pull/8196) Show deprecation warning for TAM
+* [#8208](https://github.com/timescale/timescaledb/pull/8208) Use `NULL` compression for bool batches with all null values like the other compression algorithms
+* [#8223](https://github.com/timescale/timescaledb/pull/8223) Support for Attach/Detach Chunk 
+* [#8265](https://github.com/timescale/timescaledb/pull/8265) Set incremental Continous Aggregate refresh policy on by default
+* [#8274](https://github.com/timescale/timescaledb/pull/8274) Allow creating concurrent continuous aggregate refresh policies
+* [#8314](https://github.com/timescale/timescaledb/pull/8314) Add support for timescaledb_lake in loader
+* [#8209](https://github.com/timescale/timescaledb/pull/8209) Add experimental support for Direct Compress of `COPY`
+
+**Bugfixes**
+* [#8153](https://github.com/timescale/timescaledb/pull/8153) Restoring a database having NULL compressed data
+* [#8164](https://github.com/timescale/timescaledb/pull/8164) Check columns when creating new chunk from table
+* [#8294](https://github.com/timescale/timescaledb/pull/8294) The "vectorized predicate called for a null value" error for WHERE conditions like `x = any(null::int[])`.
+* [#8307](https://github.com/timescale/timescaledb/pull/8307) Fix missing catalog entries for bool and null compression in fresh installations
+* [#8323](https://github.com/timescale/timescaledb/pull/8323) Fix DML issue with expression indexes and BHS
+
+**GUCs**
+* `enable_direct_compress_copy`: Enable experimental support for direct compression during `COPY`, default: off
+* `enable_direct_compress_copy_sort_batches`: Enable batch sorting during direct compress `COPY`, default: on
+* `enable_direct_compress_copy_client_sorted`: Correct handling of data sorting by the user is required for this option, default: off
+* `enable_optimizations`: Enable TimescaleDB query optimizations. default: on
```

Comment:
This isn't a new GUC, it exists since forever (as far I remember). Do we need to mention it here?

Reply:
Hmm, weird that it shows up now .. I did a `git diff 2.20.x..main guc.c` -- anyway I remove it! Thank you for pointing it out

Reply:
resolved via https://github.com/timescale/timescaledb/pull/8325/commits/731a97195d7929e8bbf73e18116cff90955e854e